terraform/github-to-gitlab-mirror/.envrc.private-template

```diff
@@ -0,0 +1,2 @@
+# Got to https://gitlab.com/-/profile/personal_access_tokens
+export GITLAB_TOKEN=<your_token>
```

Comment:
We could add https://github.com/nix-infra-bot as a gitlab admin and put the token in sops?

Reply:
Well the equivalent for gitlab.com, so a new account.

Reply:
> Well the equivalent for gitlab.com, so a new account.

Sorry, I don't understand?

Reply:
Ah. I didn't thought about the gitlab single-sign-on.